diff --git a/node_modules/solid-start/plugin.js b/node_modules/solid-start/plugin.js
index e9b2d3a..5b319b7 100644
--- a/node_modules/solid-start/plugin.js
+++ b/node_modules/solid-start/plugin.js
@@ -37,7 +37,7 @@ function solidStartClientAdpater() {
               manualChunks: undefined
             }
           },
-          minify: "terser"
+          minify: false
         }
       });
     }
@@ -227,6 +227,7 @@ function solidsStartRouteManifest(options) {
         build: {
           target: "esnext",
           manifest: true,
+          minify: false,
           rollupOptions: {
             plugins: [
               manifest({
